File: Part1_강의자료/Ch01_문자열/문제별코드/1543_문서검색/1543.java
import java.util.Scanner;

class Main
{
    public static void main (String[] args)
    {
        Scanner sc = new Scanner(System.in);
        String doc = sc.nextLine();
        String word = sc.nextLine();
        int startIndex = 0;
        int count = 0;
        while (true) {
            int findIndex = doc.indexOf(word, startIndex);
            if (findIndex < 0)
                break;
            startIndex = findIndex + word.length();
            count++;
        }
        System.out.println(count);
    }
}

File: Part1_강의자료/Ch01_문자열/문제별코드/1543_문서검색/1543.py
# doc=input()
# word=input()
# print(doc.count(word))

doc = input()
word = input()

start_index = 0
count = 0
while True:
    find_index = doc.find(word, start_index)
    if find_index < 0:
        break
    start_index = find_index + len(word)
    count += 1

print(count)
